pkgs/development/web/bloomrpc/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, fetchurl, appimageTools, pkgs }:
+
+let
+  pname = "bloomrpc";
+  version = "1.5.3";
+  name = "${pname}-${version}";
```

Comment:
```suggestion
```
all appimage tools now also understand pname.

Reply:
did it

Reply:
Thanks! I think we no longer need this line.